### ISSUE TITLE
Context menus do not pop up when using Voiceover keys on the Mac.  j

### ISSUE THREAD

**Issue by wxtrac** [wxWidgets/wxWidgets#13010]
*Issue migrated from trac ticket # 13010*

**component:** GUI-all | **priority:** normal | **keywords:** contexty menu voiceover speech mac

#### 2011-03-04 07:33:50: kperry@blinksoft.com (Ken Perry) created the issue

___

If you create a context menu with any version of wxwidgets you can not activate it using the voiceover keys.  Voiceover is activated by pressing cmd-f5  then if you are on a control that has a context menu you should be able to do shift-control-option-m to bring up the context menu.  The right click works but the key press does not.  This makes it impossible to bring up a context menu in a wx widget program on the mac for a blind user.  Below is the program I used to test.

```py
#!/usr/bin/python

# contextmenu.py

import wx


class MyPopupMenu(wx.Menu):
    def __init__(self, parent):
        wx.Menu.__init__(self)

        self.parent = parent

        minimize = wx.MenuItem(self, wx.NewId(), 'Minimize')
        self.AppendItem(minimize)
        self.Bind(wx.EVT_MENU, self.OnMinimize, id=minimize.GetId())

        close = wx.MenuItem(self, wx.NewId(), 'Close')
        self.AppendItem(close)
        self.Bind(wx.EVT_MENU, self.OnClose, id=close.GetId())


    def OnMinimize(self, event):
        self.parent.Iconize()

    def OnClose(self, event):
        self.parent.Close()


class ContextMenu(wx.Frame):
    def __init__(self, parent, id, title):
        wx.Frame.__init__(self, parent, id, title, size=(250, 150))

        self.Bind(wx.EVT_CONTEXT_MENU, self.OnRightDown)

        self.Center()
        self.Show()

    def OnRightDown(self, event):
        print "test"

       # self.PopupMenu(MyPopupMenu(self), event.GetPosition())


app = wx.App()
frame = ContextMenu(None, -1, 'context menu')
app.MainLoop()

class MyPopupMenu(wx.Menu):
      def __init__(self, parent):
          wx.Menu.__init__(self)
```
